toolkit/scripts/containerized-build/resources/splash.md

```diff
@@ -0,0 +1,3 @@
+# Generate splashes with:
+figlet "Mariner Builder!" |lolcat --seed 0  --force > ./resources/builder_splash.txt && cat ./resources/builder_splash.txt
```

Comment:
IMHO, adding the colors is a little tacky.

Reply:
Tagging @dmcilvaney and grabbing popcorn